L1Trigger/L1TNtuples/plugins/L1CaloTowerTreeProducer.cc

```diff
@@ -59,10 +59,10 @@ Description: Produce L1 Extra tree
 // class declaration
 //
 
-class L1CaloTowerTreeProducer : public edm::EDAnalyzer {
+class L1CaloTowerTreeProducer : public edm::one::EDAnalyzer<> {
```

Comment:
Since this is using the TFileService and only one module at a time can use that Service, this needs to be
```
class L1CaloTowerTreeProducer : public edm::one::EDAnalyzer<edm::one::SharedResources> {
```